src/openexr.imageio/exrinput.cpp

```diff
@@ -319,7 +320,9 @@ void set_exr_threads ()
 
     int oiio_threads = 1;
     OIIO::getattribute ("exr_threads", oiio_threads);
-
+    if (oiio_threads == 0) {
+        oiio_threads = Sysutil::hardware_concurrency();
```

Comment:
OK, makes sense to me. I'll merge this.

Do you think I should change the default (if the user never sets the attribute) to also be hardware_concurrency rather than the current physical_concurrency?

Have you actually benchmarked 12 vs 24 threads on your system?